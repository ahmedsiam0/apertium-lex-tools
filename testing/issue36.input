^toempty<adv>/nonempty<adv>/<adv>$
^emptier<adv>/nonempty<adv>/$
^meme<vblex><pres>/pickme<vblex><pres>/notme<vblex><pres>$
^meme<vblex><pres>/andme<vblex><pres>/notme<vblex><pres>$
