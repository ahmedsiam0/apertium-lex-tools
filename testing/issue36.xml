<rules>
  <rule c="check that we can select empty lemmas">
    <match lemma="toempty" tags="adv">
      <select lemma="" tags="adv"/>
    </match>
  </rule>
  <rule c="even empty lemmas without tags">
    <match lemma="emptier" tags="adv">
      <select lemma=""/>
    </match>
  </rule>
  <rule c="but unspecified lemmas should be equivalent to star lemmas">
    <match tags="vblex.*">
      <select lemma="pickme"/>
    </match>
  </rule>
  <rule c="so star lemmas should be equivalent to unspecified lemmas">
    <match lemma="*" tags="vblex.*">
      <select lemma="andme"/>
    </match>
  </rule>
</rules>
