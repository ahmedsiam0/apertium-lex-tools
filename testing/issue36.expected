^toempty<adv>/<adv>$
^emptier<adv>/nonempty<adv>/$
^meme<vblex><pres>/pickme<vblex><pres>$
^meme<vblex><pres>/andme<vblex><pres>$
